packages/app/src/systems/DApp/hooks/useTransactionRequest.tsx

```diff
@@ -25,15 +25,16 @@ const selectors = {
     return state.context;
   },
   isShowingInfo({
-    account,
     isLoading,
-  }: Omit<ReturnType<typeof useAccount>, 'handlers'>) {
+  }: Omit<
+    ReturnType<typeof useAccounts>,
+    'handlers' | 'accounts' | 'account'
+  >) {
     return (state: TransactionMachineState) =>
       !isLoading &&
       !state.context.approvedTx &&
       !state.context.txApproveError &&
-      state.context.origin &&
-      account;
+      state.context.origin;
```

Comment:
We could add back account here and fix on `TransactionRequest` to pass `ctx.account!`